docs/03-apps-and-frameworks.md

```diff
@@ -0,0 +1,18 @@
+
+### Laravel
+  * [Account](http://account.stjude.cloud) / [Github](https://github.com/stjudecloud/account)
+  * [Viz Community](https://viz.stjude.cloud) / [Github](https://github.com/stjude/viz)
+  * [PeCan](https://pecan.stjude.cloud) / [Github](https://github.com/stjude/pecan) / `Uses Vue for Front-End`
+  * Vanguard / [Github](https://github.com/stjudecloud/vanguard) / `In Development`
+  * Intro to Genomics / [Github](https://github.com/stjude/introduction-to-genomics-for-engineers) / `In Development`
+
+### React
+  * [Platform](https://platform.stjude.cloud) / [Github](https://github.com/stjude/genomics-platform)
+  * [St. Jude Cloud](https://stjude.cloud) / [Github](https://github.com/stjude/stjude.cloud) / `Gatsby`
+  * [Docs](https://design.stjude.cloud/docs) / [Github](https://github.com/stjudecloud/docs) / `Gatsby`
```

Comment:
https://www.stjude.cloud/docs/ - remove `design.` from URL

Reply:
And the documentation uses `mkdocs` instead of Gatsby, so we should probably move it to another section.

Reply:
Good catch! I've also added an MKDOCS section for Docs.